components/AdminPublishButton/AdminPublishButton.vue

```diff
@@ -6,14 +6,15 @@
     :external="type === 'harvester'"
   >
     <span v-if="type === 'harvester'">{{ t("Créer un moissoneur sur") }}</span>
-    <span v-else>{{ t("Publier sur") }}</span>&nbsp;<SiteLogo />
+    <span v-else>{{ t("Publier sur") }}</span>&nbsp;<LogoAsText />
```

Comment:
I think using the text only from the new logo would do?
<img width="268" height="91" alt="image" src="https://github.com/user-attachments/assets/150d07b5-f707-4f6e-b96f-91abb388ee7a" />